admin/types.d.ts

```diff
@@ -0,0 +1,43 @@
+export type ReticulumStorageT = {
+  in_quota: boolean;
+};
+
+export type ReticulumSceneListingT = {
```

Comment:
@keianhzo do you know where a good place for me to look to build these types out?

Reply:
I believe that would be the reticulum source code:
- https://github.com/mozilla/reticulum/blob/master/lib/ret/meta.ex
- https://github.com/mozilla/reticulum/blob/master/lib/ret/avatar_listing.ex
- https://github.com/mozilla/reticulum/blob/master/lib/ret/scene_listing.ex
- https://github.com/mozilla/reticulum/blob/master/lib/ret/account.ex
- https://github.com/mozilla/reticulum/blob/master/lib/ret/storage.ex

Reply:
amazing thank you

Reply:
@keianhzo ok I think this is ready to review when you get a chance - it's a big departure from the current way we do things here so let me know if you want me to walk through my future plans if something seems odd.